controllers/graphController.js

```diff
@@ -141,6 +140,12 @@ GraphController.prototype.embed = function(req, res, next) {
 	}).catch(next)
 }
 
+// "moon-test-one" -> "Moon Test One"
+function convNameToHumanReadable(name) {
```

Comment:
How ? convGraphNameToHumanReadable(graphName) ?
filterGraphNameForTitle?

What is your preference ? :)